### ISSUE TITLE
Package task, path fails - includes bower and node

### ISSUE THREAD

**Issue by erwstout** [factor1/prelude-wp#77]
The default path for the package task fails when ran. Source should be `'./**/*'` instead of what it is currently. 

Also, when packaging it includes `bower_components/` and `node_modules/` these should be ignored.

**Comment by erwstout** [comment]
```js
return gulp.src( [
    './**/*',
    '!bower_components',
    '!node_modules',
    '!bower_components/**',
    '!node_modules/**'
  ] )
```

**Comment by erwstout** [comment]
Fixed in `3.3.6`
